cclib/parser/turbomoleparser.py

```diff
@@ -123,6 +123,34 @@ def extract(self, inputfile, line):
                 revision = tokens[2]
                 self.metadata["package_version"] = "{}.r{}".format(package_version, revision)
 
+
+        ## Basis set info from dscf.
+        #               +--------------------------------------------------+
+        #               |               basis set information              |
+        #               +--------------------------------------------------+
+        # 
+        #               we will work with the 1s 3p 5d 7f 9g ... basis set
+        #               ...i.e. with spherical basis functions...
+        # 
+        #    type   atoms  prim   cont   basis
+        #    ---------------------------------------------------------------------------
+        #     o        1     15      5   sto-3g hondo  [2s1p|6s3p]
+        #     h        2      3      1   sto-3g hondo  [1s|3s]
+        #    ---------------------------------------------------------------------------
+        if "type   atoms  prim   cont   basis" in line:
+            line = next(inputfile)
+            line = next(inputfile)
+            basis_sets = []
+            while "---------------------------------------------------------------------------" not in line:

```

Comment:
Since the barrier is only dashes, you can do a thing we do in other places that looks like
```python
while set(line.strip()) != {"-"}:
```

Reply:
cool, thanks for the tip